docker/multi-process/Dockerfile

```diff
@@ -1,32 +1,24 @@
 FROM ubuntu:14.04
 
-ADD docker/scripts/prepare /scripts/prepare
+COPY docker/scripts/prepare /scripts/
 RUN /scripts/prepare
 
-ADD docker/multi-process/scripts/standalone-packages /scripts/standalone-packages
+COPY docker/multi-process/scripts/standalone-packages /scripts/
 RUN /scripts/standalone-packages
 
 WORKDIR /app
-
-ADD ["Gemfile", "Gemfile.lock", "/app/"]
-ADD lib/gemfile_helper.rb /app/lib/
-ADD vendor/gems /app/vendor/gems
-
-RUN chown -R huginn:huginn /app && \
-    sudo -u huginn -H echo "gem 'sqlite3', '~> 1.3.11'" >> /app/Gemfile && \
-    sudo -u huginn -H LC_ALL=en_US.UTF-8 RAILS_ENV=production ON_HEROKU=true bundle install --without test development --path vendor/bundle -j 4
-COPY . /app
+COPY ./ /app/
 
 ARG OUTDATED_DOCKER_IMAGE_NAMESPACE=false
 ENV OUTDATED_DOCKER_IMAGE_NAMESPACE ${OUTDATED_DOCKER_IMAGE_NAMESPACE}
 
-ADD ["docker/scripts/setup", "docker/scripts/setup_env", "docker/multi-process/scripts/init", "/scripts/"]
-
+COPY docker/scripts/setup /scripts/
 RUN /scripts/setup
 
-VOLUME /var/lib/mysql
```

Comment:
Why was this removed?

Reply:
This was a hack. I think I just thought of a better way. I understand it's a good practice to explicitly expose the database storage as a volume. The problem was when Kubernetes assigns a pod to a node, it creates an `emptyDir` for each `VOLUME` directive with an ownership `root:$USER` that `mysql_install_db` will try (and ultimately fail) to change to `$USER:$USER`. It's not necessary if we want to run mysql as `$USER`. Do you know how to ask `mysql_install_db` to skip changing ownership of the `datadir`? I'm thinking to initialize the `datadir` in `/tmp/mysql`, and move everything to /var/lib/mysql. This shouldn't be a big deal since it's only done the first time the database is initialized.

Reply:
I don't know if that is possible in `mysql_install_db`. Do you think it is required to support that use case on kubernetes? If one runs a kubernetes cluster wouldn't it be safer to link to an external database and/or use the `single-process` image?

Reply:
It should be fine now.